Cargo.toml

```diff
@@ -42,10 +42,11 @@ ring = "0.17"
 serde = { version = "1.0.204", features = ["derive"] }
 serde_json = { version = "1.0.117", features = ["float_roundtrip"] }
 thiserror = "1.0.63"
-uniffi = { git = "https://github.com/mozilla/uniffi-rs", features = [
-    "tokio",
-    "cli",
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", branch = "main", features = [
```

Comment:
Do we know what the uniffi release schedule looks like right now? It would be nice to get on a released version so Rust can release as well

Reply:
They're working up to a breaking release that would have what we need in it. What I should really do here for now is pin to a hash.

Reply:
Ah, I can't pin to a hash directly. Cargo treats a pinned rev version as different than a `main` requirement locked to the same rev, so the tests won't build (different perceived traits between Java and the others). 

I'll try to see if the patch version they just released has what we need, but I don't think it does. So we'll be on main still till they release.